docs/gsod/2024/proposal.md

```diff
@@ -0,0 +1,154 @@
+---
+seo:
+  title: Redocly proposal - Google Season of Docs 2024
+slug: /gsod-2024-proposal
+theme:
+  feedback:
+    hide: true
+---
+
+# Google Season of Docs 2024 - Redocly proposal
+
+[Google Season of Docs](https://developers.google.com/season-of-docs) provides support for open-source projects to improve their documentation and gives professional technical writers an opportunity to gain experience in open source.
+Together we raise awareness of open source, of docs, and of technical writing.
+
+Redocly is applying to participate in Google Season of Docs 2024.
+
+## About Redocly
+
+[Redocly](https://redocly.com/) was founded in 2017 with the mission to get more consumers using APIs with less hands-on support.
+We were born out of [Redoc](https://redocly.com/docs/redoc/), the popular open-source OpenAPI documentation software project used by thousands of companies worldwide.
+We make API design and documentation software with the goal to accelerate API ubiquity.
+Our suite of products makes our customers’ APIs more accessible and loved by their end users and consumers.
+
+## How to apply
+
+We are glad you are interested in contributing to Redocly's open source project documentation.
+Please send your [statement of interest](https://developers.google.com/season-of-docs/docs/tech-writer-statement) to [gsod@redocly.com](mailto:docs@redocly.com) by April 26th.
+
+Be sure to review the skills needed for each project and include references to your abilities in these areas in particular.
+
+You also need to be able to meet remotely within the following timezones:
+
+- Eastern Standard Time (EST) - UTC-05:00
+- Greenwich Mean time (GMT) - UTC+00:00
```

Comment:
Does anyone really know what UK summer time is though? (BST). I'd handwave this and say "US east coast and EU timezones".

Reply:
ok, will update